src/containers/editInput/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useContext, useState, useRef } from "react";
+import { ToDoContext } from "../app/context";
+import  useFocus  from "../../hooks/useFocus";
+import "./index.sass";
+
+const InputEdit = ({ task, setEnableEdit }) => {
+  const [newValue, setNewValue] = useState("");
```

Comment:
I think here should be an initialValue that equals to editable task value 
const [newValue, setNewValue] = useState(() => task.text);